src/backend/base/langflow/main.py

```diff
@@ -208,12 +208,12 @@ async def flatten_query_string_lists(request: Request, call_next):
     @app.exception_handler(Exception)
     async def exception_handler(request: Request, exc: Exception):
         if isinstance(exc, HTTPException):
-            logger.error(f"HTTPException: {exc.detail}")
+            logger.error(f"HTTPException: {exc}", exc_info=exc)
```

Comment:
Using `exc` instead of `exc.detail` adds the error code
```pyconsole
>>> from fastapi import HTTPException
>>> str(HTTPException(status_code=404, detail="Item not found"))
'404: Item not found'
```

Reply:
Cool